packages/web3-core/types/index.d.ts

```diff
@@ -44,10 +46,13 @@ export class AbstractWeb3Module {
     transactionPollingTimeout: number;
     defaultGasPrice: string;
     defaultGas: number;
+    static readonly providers: Providers;
     readonly providers: Providers;
     defaultAccount: string | null;
     readonly currentProvider: EthereumProvider | HttpProvider | IpcProvider | WebsocketProvider;
+    static readonly currentProvider: EthereumProvider | HttpProvider | IpcProvider | WebsocketProvider;
```

Comment:
the currentProvider property isn't static.

Reply:
Yeah your correct remove it 👍